src/openforms/registrations/contrib/objects_api/tests/test_template.py

```diff
@@ -16,17 +15,42 @@
     SubmissionFileAttachmentFactory,
 )
 from openforms.submissions.tests.mixins import SubmissionsMixin
+from openforms.utils.tests.vcr import OFVCRMixin
 
 from ..models import ObjectsAPIConfig
 from ..plugin import PLUGIN_IDENTIFIER, ObjectsAPIRegistration
+from ..typing import RegistrationOptionsV1
 
+TEST_FILES = Path(__file__).parent / "files"
 
-class JSONTemplatingTests(TestCase):
+
+class JSONTemplatingTests(OFVCRMixin, TestCase):
     maxDiff = None
```

Comment:
I guess we do not need that but it's ok if we do not have to change anything else.

Reply:
oh I definitely needed it while adapting the tests to see the full diff! there was quite a bit wrong in the original assertions